src/js/core/widget/core/Tabs.js

```diff
@@ -204,9 +204,12 @@
 			 */
 			prototype._initHeight = function () {
 				var self = this,
-					page = ns.widget.Page(self._ui.page);
+					page;
 
-				self.element.style.height = page.getContentHeight() + "px";
+				if (!self.element.style) {
```

Comment:
Is it correct condition?
If the 'self.element.style' is 'undefined', I think we cannot set the 'height' value.

Reply:
Thank you for review. I did mistake in this place. Purpose of this change is remain height if the height was defined by inline styles by developer.